rel-eng/Makefile.git

```diff
@@ -1,85 +0,0 @@
-
-THIS_MAKEFILE := $(realpath $(lastword $(MAKEFILE_LIST)))
```

Comment:
Still referenced in : client/tools/mgr-push/Makefile

Reply:
This one can go away too: not used by the spec file and only containing this include